wandb/sdk/wandb_run.py

```diff
@@ -1,8 +1,6 @@
 #
 # -*- coding: utf-8 -*-
```

Comment:
probably can remove this one as well... not needed in python 3 since the default encoding is already utf-8